app/controllers/concerns/defendant_helpers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module DefendantHelpers
```

Comment:
I'd rename this since you now have two DefendantHelper(s) modules - DefendantSearchable perhaps.

TBH i don't think we should be using a concern - a service object would be better as it can be tested easily in isolation and is not so tightly coupled.

Reply:
I have renamed defendant_helper to defendant_searchable 👍 

Regarding your comment about using service object rather than concern, I take your point about not being able to test the helper in isolation, however, I introduced the helper in order to avoid duplicating some shared code and I am not sure in my own mind how a service object would work in this instance. Would you be happy to have a separate spike ticket to investigate using a service object perhaps this could also include the work to remove dependencies on defendant asn and nino?

Reply:
yes, we should do this. can you add it as a checklist item on the existing ticket for stopping reliance or asn and nino, and link that to a separate ticket to break it into a service object (as we may or may not get time to do that)